### DISCUSSION TITLE
update derzeit nicht möglich

### DISCUSSION THREAD

**Discussion by hghgsat** [brutella/hkknx-public#251] [General]
Fehler
Get "https://api.github.com/repos/brutella/hkknx-public/releases": x509: certificate has expired or is not yet valid: current time 2019-02-14T10:17:26Z is before 2022-03-16T00:00:00Z

**Comment by hghgsat** [comment]
[Hilfe](https://hochgatterer.me/hkknx/docs) • [Konsole](http://192.168.0.205:8080/system/log) • [Sprache](http://192.168.0.205:8080/update/check#)
© Matthias Hochgatterer •   • [2.4.1 ](http://192.168.0.205:8080/update/check#)(2022-09-13T13:27:07+02:00)

**Comment by brutella** [comment]
Bitte Datum und Uhrzeit aktualisieren → siehe https://hochgatterer.me/de/hkknx/docs/help/date-and-time/

**Comment by hghgsat** [comment]
Hi,

danke – habs Datum gesetzt, dann dürfte die DB ein Problem gehabt haben, jedenfalls wollte er ein neues Projekt etc. anlegen.
Musste ein Backup zurückspielen, jetzt klappts wieder. Bin im Linux leider nicht wirklich sattelfest, konnte mir nur so helfen.

Danke
Harry


Von: Matthias ***@***.***>
Gesendet: Freitag, 13. Jänner 2023 17:57
An: brutella/hkknx-public ***@***.***>
Cc: HGS: Harald Gergits ***@***.***>; Author ***@***.***>
Betreff: Re: [brutella/hkknx-public] update derzeit nicht möglich (Discussion #251)


Bitte Datum und Uhrzeit aktualisieren → siehe https://hochgatterer.me/de/hkknx/docs/help/date-and-time/
